doc/api/worker_threads.md

```diff
@@ -335,7 +339,8 @@ In particular, the significant differences to `JSON` are:
 * `value` may contain typed arrays, both using `ArrayBuffer`s
    and `SharedArrayBuffer`s.
 * `value` may contain [`WebAssembly.Module`][] instances.
-* `value` may not contain native (C++-backed) objects other than `MessagePort`s.
+* `value` may not contain native (C++-backed) objects other than `MessagePort`s
+  and [`FileHandle`][]s.
```

Comment:
In the future, we might want to consider marking transferable objects with a special static read-only symbol property so we don't actually have to maintain this list. For instance, something like:

```js
const { transferableSymbol } = require('worker_threads');

MessagePort[transferableSymbol]         // True!
FileHandle[transferableSymbol]          // True!
MyOtherHandleThing[transferableSymbol   // False!
```

Then we wouldn't necessarily have to maintain a list of such objects in the docs, we would just say that `value` may be any object where `[transferableSymbol] === true`

Reply:
@jasnell I think that’s a good idea, but I do think it’s important to list the built-in types that are transferable here.

Reply:
Agreed, but I think we could put those in one place rather than having them listed in a couple different places (they're currently listed in the `value` and the `transferList`, etc)